main.cpp

```diff
@@ -45,6 +52,44 @@ int main() {
                 .build();
     });
 
+    server.register_handler<HttpMethod::POST>("/search/{id, ip_address}", [](const HttpRequestContext &context) {
```

Comment:
This is an invalid path parameter. Do not copy-paste things from docs...

Reply:
I did not copy-paste from the docs. This was the example you gave me on how to write path parameters a while back when i was at your place. Seeing as I still do not have an example I need to ask you what is the correct way to do it. I would guess it would look sth like this 
[id, ip_address](const HttpRequestContext &context) and after that i would access them through context.get_path_params(path_elem)